docs/user-guide/admin-cli.md

```diff
@@ -39,17 +39,17 @@ optional arguments:
 ### Create
 A tenant named "_DEFAULT" will be created automatically post install.
 
-Creates a new named tenant and optionally assigns VMs.
+Creates a new named tenant and optionally assigns VMs. Valid tenant name is only allowed to be "[a-zA-Z0-9_][a-zA-Z0-9_.-]*"
```

Comment:
This tenant name validation fix is not related to issue 816 itself.  Sam pointed out earlier that in his comment to suggest that we should give some limitation to the tenant name during creation. For example, no space allowed in the tenant name, which will avoid some potential issues that may occur like issue 816 if user specify tenant name with spaces and other weird characters.